assets/router/deployment.yaml

```diff
@@ -18,6 +18,9 @@ spec:
         - name: router
           # image is set at runtime.
           imagePullPolicy: IfNotPresent
+          securityContext:
+            # See https://bugzilla.redhat.com/2007246
```

Comment:
with bugzilla going away, you may not want to leave links in your code

Reply:
I'd like to keep this reference for the sake of providing context; I believe Bugzilla will remain in read-only mode at least for a period of time, and this reference can still be found in JIRA via the BZ ID.